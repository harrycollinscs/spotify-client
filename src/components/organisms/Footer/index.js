import React from "react";
import styled from "styled-components";

const Container = styled.div`
  width: 100vw;
  background-color: #212121;
  bottom: 0;
  height: 20vh;
  margin: 0px -30px;
`;

const Footer = () => {
  return (
    <Container>

    </Container>
  );
};

export default Footer;
