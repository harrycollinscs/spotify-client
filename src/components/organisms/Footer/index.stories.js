import React from "react";
import Footer from "./index";

export default {
  title: "Organisms/Footer",
  component: Footer,
};

const Template = (args) => <Footer {...args} />;

export const Default = Template.bind({});
