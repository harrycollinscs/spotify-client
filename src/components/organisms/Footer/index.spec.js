import React from "react";
import Footer from "./index";
import { shallow } from "enzyme";

describe("Organisms/Footer", () => {
  const component = <Footer />;

  it("Should match snapshot", () => {
    const render = shallow(component);
    expect(render).toMatchSnapshot();
  });
});
