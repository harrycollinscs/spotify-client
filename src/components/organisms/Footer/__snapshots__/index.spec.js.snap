// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Organisms/Footer Should match snapshot 1`] = `<styled.div />`;
